src/main/java/io/stargate/sgv2/jsonapi/exception/UpdateException.java

```diff
@@ -0,0 +1,31 @@
+package io.stargate.sgv2.jsonapi.exception;
+
+/**
+ * Errors related to the filter clause in a request.
+ *
+ * <p>See {@link APIException} for steps to add a new code.
+ */
+public class UpdateException extends RequestException {
+
+  public static final Scope SCOPE = Scope.UPDATE;
+
+  public UpdateException(ErrorInstance errorInstance) {
+    super(errorInstance);
+  }
+
+  public enum Code implements ErrorCode<UpdateException> {
+    UPDATE_UNKNOWN_TABLE_COLUMNS,
```

Comment:
use UNKNOWN_TABLE_COLUMNS same as for the filter and document scopes, and add newlines ot the error messahes to be the same.

Reply:
fixed.